app/src/main/java/tech/jpco/qen/view/QenPage.kt

```diff
@@ -0,0 +1,154 @@
+package tech.jpco.qen.view
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Paint
+import android.util.AttributeSet
+import android.util.Log
+import android.view.MotionEvent
+import android.view.View
+import androidx.core.content.res.ResourcesCompat
+import com.jakewharton.rxbinding3.view.layoutChanges
+import com.jakewharton.rxbinding3.view.touches
+import io.reactivex.Observable
+import tech.jpco.qen.R
+import tech.jpco.qen.viewModel.DrawPoint
+import tech.jpco.qen.viewModel.TAG
+import tech.jpco.qen.viewModel.TouchEventType
+import java.util.concurrent.TimeUnit
+
+class QenPage @JvmOverloads constructor(
+    context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
+) : View(context, attrs, defStyleAttr) {
+    private var mHeight = 0
+    private var mWidth = 0
```

Comment:
Since these are set in `onSizeChanged, I think you can get away with just referencing the `height` and `width` properties rather than having these private variables.

Reply:
You're probably right, which means I feel clever...